targets/ingester/src/transform/agreements/getKaliBlock.ts

```diff
@@ -1,17 +1,19 @@
 import { client } from "@shared/graphql-client";
-
+import { KaliArticleHDN } from "@shared/types";
```

Comment:
Prefer `import type`

```suggestion
import type { KaliArticleHDN } from "@shared/types";
```

Reply:
Just curious, what is the there a difference wihtout ?

Reply:
With `type` it'll just import the definition, it'll be removed at the runtime. It has been added to typescript for babel error or circular dependencies errors (https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html). It's a good practice to split type to help the transpiler or analysis tools. I'd never use it but it seems to be useful in specific case but @douglasduteil have perhaps another feedback.

Reply:
ok thanks !